cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/compound_outlet.json

```diff
@@ -0,0 +1,75 @@
+{
+  "project-id": {
+    "office-id": "SPK",
+    "name": "location"
```

Comment:
We don't need to fix this right now, but let's start using more natural names for location, elements, etc.

I had someone read a test to get a sense of usage and they got confused because location name was basically the test name.

Where that doesn't work within a test, like we need really specific names, make let's make sure to use an obvious variable name within the code.

Reply:
Oh that's a good point.  Thank you for bringing this up, Mike, I'll do that moving forward.